k8s-deployment.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pass-docker-reg
+  labels:
+    app: eclipse-pass
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: eclipse-pass
+  template:
+    metadata:
+      labels:
+        app: eclipse-pass
+    spec:
+      containers:
+      - name: pass-deposit-services
+        image: ghcr.io/eclipse-pass/deposit-services:v0
+      - name: pass-static-html
+        image: ghcr.io/eclipse-pass/static-html:v0
+        ports:
+        - containerPort: 80
+          name: static-html
+      - name: pass-activemq
+        image: ghcr.io/eclipse-pass/activemq:v0
+        ports:
+        - containerPort: 61616
+          name: activemq-jms
+        - containerPort: 61613
+          name: activemq-stomp
+        - containerPort: 8161
+          name: activemq-web
+      - name: pass-indexer
+        image: ghcr.io/eclipse-pass/indexer:v0
+      - name: pass-postgres
+        image: ghcr.io/eclipse-pass/postgres:v0
+        ports:
+        - containerPort: 5432
+          name: postgres-db
+      - name: pass-ldap
+        image: ghcr.io/eclipse-pass/ldap:v0
+      - name: pass-ftpserver
+        image: ghcr.io/eclipse-pass/ftpserver:v0
+        ports:
+        - containerPort: 30000
+          name: ftpserver
+      - name: pass-idp
+        image: ghcr.io/eclipse-pass/idp:v0
+      - name: pass-doi-service
+        image: ghcr.io/eclipse-pass/doi-service:v0
+        ports:
+        - containerPort: 8090
+          name: http-web-svc
+      - name: pass-schema-service
+        image: ghcr.io/eclipse-pass/schema-service:v0
+        ports:
+        - containerPort: 8086
+          name: schema-service
+      - name: pass-policy-service
+        image: ghcr.io/eclipse-pass/policy-service:v0
+        ports:
+        - containerPort: 8088
+          name: policy-service
+      - name: pass-download-service
+        image: ghcr.io/eclipse-pass/download-service:v0
+        ports:
+        - containerPort: 6502
+          name: download-svc
+      # - name: pass-ember
```

Comment:
These containers are failing for a variety of reasons. I will add them to the configuration as I resolve the issues with each of them.